Add AnyLogic.app v7.1.2

denpatin opened the pull request:
Great tool! I'm suprised why it wasn't present as a cask!

AnyLogic is:
 > * A multimethod simulation software
 > * The only simulation tool that supports Discrete Event, Agent Based, and System Dynamics Simulation

There are several versions of this software but I have chosen the `Personal Learning Edition`, which is free for personal & self-educational use. If needed, I can also add `commercial` versions, but I am not sure how to name the casks for them.

vitorgalvao commented on Casks/anylogic.rb:
```diff
@@ -0,0 +1,25 @@
+
+  caveats <<-EOS.undent
+    For activation instructions, check up on
+      http://www.anylogic.com/upload/activation-guides/AnyLogic_#{version.slice(/\w+/)}_PLE_License.pdf
```
`#{version.slice(/\w+/)}` can be greatly simplified as `#{version.to_i}`.

vitorgalvao commented:
It’s missing `uninstall`. There are [scripts](https://github.com/phinze/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/phinze/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that. Do you need any help?

Also, please unify the caveats.

denpatin commented:
@vitorgalvao Thank you for help!

* Simplified the code you indicated
* Added the `uninstall` section
* Unified the caveats

And rebased the commit.

vitorgalvao commented:
Looks good. Thank you.
